src/scripts/utils/translations/RawTranslatedText.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import { renderToString } from 'react-dom/server'
+
+import TranslatedText from 'components/translations/TranslatedText'
+
+const RawTranslatedText = props => renderToString(<TranslatedText {...props} />)
+
+export default RawTranslatedText
```

Comment:
This is needed when we have to pass translations to html element attributes, e.g. `placeholder` for an `input`